www/components/MeetingNameDate.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable indent */
+/* eslint-disable no-trailing-spaces */
+// David
+// This component has a flag icon which is white while meeting
+// is being edited and red when saved,
+// a meeting title and a date
+
+import styles from "../styles/MeetingNameDate.module.css";
+import Input from "./Input";
+import FlagIcon from "@material-ui/icons/Flag";
+
+
+function MeetingNameDate() {
+  return (
+    <div className={styles.meetingBar}>
+      <FlagIcon style={{ color: "var(--agendaSecondary)", fontSize: 37 }}/>
+      <form className={styles.form}>
+          <div>
+            <Input placeholder="Meeting Name" />
+          </div>
+          <div className={styles.date}>
+            <Input placeholder="mm/dd/yy hh:mm"  />
+          </div>
```

Comment:
Instead of wrapping the `Input` components in `div`s like this you can select the `Input` in the CSS like this because it is within the `form` class:


```
.form Input {
  mystyles;
}
```

Reply:
I have two inputs and i want each one styled differently because of the spacing